packages/agoric-cli/src/commands/auction.js

```diff
@@ -89,16 +89,12 @@ export const makeAuctionCommand = (
        * }} opts
        */
       async opts => {
-        const { agoricNames, readLatestHead } = await makeRpcUtils(
+        const { agoricNames, readPublished } = await makeRpcUtils(
           { fetch },
           networkConfig,
         );
 
-        /** @type {{ current: AuctionParamRecord }} */
-        // @ts-expect-error XXX should runtime check?
```

Comment:
This isn't a runtime regression either, but we're losing a bit of documentation. **suggestion**: add something in the `readPublished` docstring about returning a static type that isn't runtime guaranteed.